recite/admin.py

```diff
@@ -169,6 +169,6 @@ def get_segments_dict(self, encoding, segments_file):
                 ayah_number = int(ayah_str)
             except ValueError:
                 print(f"Not normal format for csv at row {row}")
-            segments_dict[surah_number][ayah_number] = segments_string
-
+            else:
```

Comment:
I rarely use `else` in exceptions, but it looks fine.
I just understood what was the problem.

Since we found a bug, let's add a unit test (or couple of them).